networkit/matching.pyx

```diff
@@ -20,39 +20,144 @@ cdef class Matching:
 		return self
 
 	def match(self, node u, node v):
+		"""
+		Set two nodes u and v as each others matching partners.
+		
+		Parameters:
+		-----------
+		u : node
+		v : node
+		"""
 		self._this.match(u,v)
 
 	def unmatch(self, node u,  node v):
+		"""
+		Reset the two nodes u and v to unmatched.
+		
+		Parameters:
+		-----------
+		u : node
+		v : node
+		"""
 		self._this.unmatch(u, v)
 
 	def isMatched(self, node u):
+		"""
+		Check if node u is matched.
+		
+		Parameters:
+		-----------
+		u : node
+		
+		Returns:
+		-----------
+		bool
+			true if u has a matching partner.
+		"""
 		return self._this.isMatched(u)
 
 	def areMatched(self, node u, node v):
+		"""
+		Check if the nodes u and v are matched to each other.
+		
+		Parameters:
+		-----------
+		u : node
+		v : node
+		
+		Returns:
+		-----------
+		bool
+			true if node u and v are matched to each other.
+		"""
 		return self._this.areMatched(u,v)
 
 	def isProper(self, Graph G):
+		"""
+		Check whether this is a proper matching in the graph, i.e. there is an edge between
+		each pair and if the matching partner of v is u then the matching partners of u is v.
+		
+		Parameters:
+		-----------
+		G : Graph
+			Graph to be checked.
+		Returns:
+		-----------
+		bool
+			true if this is a proper matching.
+		"""
 		return self._this.isProper(G._this)
 
 	def size(self, Graph G):
+		"""
+		Get the number of edges in this matching.
+		
+		Parameters:
+		-----------
+		G : Graph
+		
+		Returns:
+		-----------
+		count
```

Comment:
`int`? There is no `count` in python.

Reply:
If you check graph.pyx for example count is used.

Reply:
the same applies there, I don't think it makes sense to mention a type that does not exist.